infoschema/perfschema/tables.go

```diff
@@ -14,10 +14,19 @@
 package perfschema
 
 import (
+	"fmt"
 	"github.com/pingcap/parser/model"
 	"github.com/pingcap/tidb/infoschema"
+	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/meta/autoid"
+	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/table"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/stmtsummary"
+)
+
+const (
+	tableNameEventsStatementsSummaryByDigest = "events_statements_summary_by_digest"
```

Comment:
I am not sure if we should resurrect perf_schema, maybe we can put it to info_schema?

Reply:
Some table names are the same, like tables.go.
And performance schema is independent with information schema in MySQL.
So I prefer the directory to be like this:
- virtual table
--infoschema
--perfschema
--sys

But now, I prefer not to change the directory structure in this PR. May fix it in another PR is better, if needed.

Reply:
Some table names are the same, like tables.go.
And performance schema is independent with information schema in MySQL.
So I prefer the directory to be like this:
virtual table
--infoschema
--perfschema
--sys

But now, I prefer not to change the directory structure in this PR. May fix it in another PR is better, if needed.